pkgerrors/stacktrace_test.go

```diff
@@ -27,6 +27,82 @@ func TestLogStack(t *testing.T) {
 	}
 }
 
+func TestLogMultiStack(t *testing.T) {
+	zerolog.ErrorStackMarshaler = MarshalMultiStack
+
+	out := &bytes.Buffer{}
+	log := zerolog.New(out)
+
+	err := errors.Wrap(errors.New("error message"), "from error")
+	log.Log().Stack().Err(err).Msg("")
+
+	got := out.String()
+	want := `\{"stack":"\[\{\"stacktrace\":\[\{\"source\":\"stacktrace_test.go\",\"line\":\"36\",\"func\":\"TestLogMultiStack\"\},.*\{\"stacktrace\".*\],"error":"from error: error message"\}`
```

Comment:
This test isn't passing right now because the json.Marshal is returning a string and I haven't quite cracked how to make the regexp match (got & want look like they should match)... The best thing to do though is to use the proper zerolog encoding rather than using json.Marshal.

Reply:
Fixed